### PR TITLE
Reformat losses task

### PR DESCRIPTION
**Pull request by tobaadesugba**
cross_entropy #1199
reformat losses #777 
reformat losses (opened by me) #2277 

I couldn't work on the docstring examples because I was not able to test them due to an issue I faced with the implementation of ivy loss functions.
Find more details in the discord losses channel [https://discord.com/channels/799879767196958751/1000044049333485648/1004362200321896548]

### CODE REVIEW

**Comment by tobaadesugba** [comment]
Also, pre-commit has been giving me problems on windows so I couldn't do pre-commit checks on it. you find the discord discussion here: https://discord.com/channels/799879767196958751/982725464110034944/1003098304592228402
